### PR TITLE
DT-72: added mock pdf generator health endpoint

### PR DESCRIPTION
**Pull request by simon-mitchell**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -56,6 +56,10 @@ app.get('/communityapi/health', (req, res) => {
   res.send({ status: 'UP' })
 })
 
+app.get('/pdfgernerator/health', (req, res) => {
+  res.send({ status: 'UP' })
```
Comment (bell-pa) [inline app.js]: Typo?

**Review by bell-pa — DISMISSED** [review]
You seem to have gernerated a typo...

File: app.js (hunk 1 above)
Comment (petergphillips) [reply app.js]: Tpyo

Comment (simon-mitchell) [reply app.js]: corrected typo

**Review by petergphillips — APPROVED** [review]

**Pull request merged**